demos/common/cpp/models/src/detection_model_retinaface.cpp

```diff
@@ -0,0 +1,389 @@
+/*
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
```suggestion
// Copyright (C) 2020-2021 Intel Corporation
```
These are nearly added files

Reply:
ok